code/game/mecha/combat/five_stars.dm

```diff
@@ -0,0 +1,22 @@
+/obj/mecha/combat/five_stars
+	desc = "A state of the art tank deployed by the Spinward Stellar Coalition National Guard."
+	name = "\improper Tank"
```

Comment:
it's not improper, you climb into the tank.

Reply:
ye, will fix

Reply:
Also, tank shouldn't be capitalized.